### ISSUE TITLE
ElasticsearchExporterException: Failed to flush bulk

### ISSUE THREAD

**Issue by MiguelPires** [zeebe-io/zeebe#5715]
**Describe the bug**
The ElasticSearch exporter fails repeatedly to flush because the connection to elasticSearch is reset. If this is sometimes expected, then we can lower the log level.

**To Reproduce**
Link: https://console.cloud.google.com/errors/CJHzrs64j9WNzAE?service=zeebe&time=P7D&project=camunda-cloud-240911&authuser=1

**Log/Stacktrace**
Logs: https://console.cloud.google.com/logs/query;query=logName:%22stdout%22%0Aresource.type%3D%22k8s_container%22%0Aresource.labels.cluster_name%3D%22production%22%0Aresource.labels.namespace_name%3D%228b3cf719-9734-468e-b38a-f1ad6ef8dec3-zeebe%22%0Aresource.labels.container_name%3D%22zeebe%22;timeRange=2020-10-25T05:33:14.522Z%2F2020-10-25T06:33:14.522Z;pinnedLogId=2020-10-25T05:36:08.399226Z%2F9obz5sxxx6itvxujm?authuser=1&project=camunda-cloud-240911

```
io.zeebe.exporter.ElasticsearchExporterException: Failed to flush bulk
	at io.zeebe.exporter.ElasticsearchClient.flush(ElasticsearchClient.java:137) ~[zeebe-elasticsearch-exporter-0.23.7.jar:0.23.7]
	at io.zeebe.exporter.ElasticsearchExporter.flush(ElasticsearchExporter.java:125) ~[zeebe-elasticsearch-exporter-0.23.7.jar:0.23.7]
	at io.zeebe.exporter.ElasticsearchExporter.flushAndReschedule(ElasticsearchExporter.java:112) ~[zeebe-elasticsearch-exporter-0.23.7.jar:0.23.7]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.23.7.jar:0.23.7]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.23.7.jar:0.23.7]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:115) [zeebe-util-0.23.7.jar:0.23.7]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.23.7.jar:0.23.7]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.23.7.jar:0.23.7]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:195) [zeebe-util-0.23.7.jar:0.23.7]
Caused by: java.io.IOException: Connection reset by peer
	at org.elasticsearch.client.RestClient$SyncResponseListener.get(RestClient.java:964) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestClient.performRequest(RestClient.java:233) ~[elasticsearch-rest-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1764) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1734) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1696) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at org.elasticsearch.client.RestHighLevelClient.bulk(RestHighLevelClient.java:472) ~[elasticsearch-rest-high-level-client-6.8.9.jar:6.8.9]
	at io.zeebe.exporter.ElasticsearchClient.exportBulk(ElasticsearchClient.java:151) ~[zeebe-elasticsearch-exporter-0.23.7.jar:0.23.7]
	at io.zeebe.exporter.ElasticsearchClient.flush(ElasticsearchClient.java:134) ~[zeebe-elasticsearch-exporter-0.23.7.jar:0.23.7]
	... 8 more
Caused by: java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:?]
	at org.apache.http.impl.nio.reactor.SessionInputBufferImpl.fill(SessionInputBufferImpl.java:231) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.codecs.AbstractMessageParser.fillBuffer(AbstractMessageParser.java:136) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:241) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39) ~[httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:114) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591) ~[httpcore-nio-4.4.13.jar:4.4.13]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
```

**Environment:**
 - OS: Camunda Cloud
 - Zeebe Version: 0.23.6
 - Configuration: [e.g. exporters etc.]

**Comment by npepinpe** [comment]
I think there is already an issue to not log exporter errors as errors, but as warnings. The reasoning was that these are not fatal to Zeebe (to some extent), and exporter authors should handle and log the critical cases themselves.

**Comment by npepinpe** [comment]
See https://github.com/zeebe-io/zeebe/issues/5564

**Comment by MiguelPires** [comment]
Closed in favor of #5564
